content/docs/release-notes/archive/_index.md

```diff
@@ -11,9 +11,10 @@ Looking for older versions of release notes for **Open 3D Engine (O3DE)**? Look
 
 | Version     |  Link                                           | Date of Release    |
 |-------------|-------------------------------------------------|--------------------|
-| 2107.1      | [2111.1 Release Notes](./21-11-release-notes)     | 12/2/2021          |
+| 2111.2      | [2111.2 Release Notes](./2111-2-release-notes)  | 1/27/2022 |
+| 2111.1      | [2111.1 Release Notes](./21-11-release-notes)     | 12/2/2021          |
 | 2107.1      | [2107.1 Release Notes](./2107-1-release-notes)   | 7/6/2021           |
```

Comment:
These dates of release may be confusing or misinterpreted by people who are used to DD/MM/YYYY format. To be more inclusive, consider switching to something more explicit, like "6 Jul 2021" and "2 Dec 2021".